### DISCUSSION TITLE
Regarding latest commit and translations

### DISCUSSION THREAD

**Discussion by Caesarovich** [localsend/localsend#1314] [Discussion (English, International)]
Hi, I didn't want to bloat issues so I'm asking here.

The [latest commit](https://github.com/localsend/localsend/commit/603c8f9625ee8cfd0287600c79ea6ac988348a30) by @harriseldon introduced a new i18n string for his zoom feature. But the PR doesn't come with the generated dart files (`app/lib/gen/strings_*.g.dart`). This causes a build error because the property is missing for the dart compiler: 

```

lib/widget/dialogs/zoom_dialog.dart:33:29: Error: The getter 'zoom' isn't defined for the class '_StringsDialogsEn'.
```

It isn't a big deal, I only have to run `dart run slang` to generate the files locally. But when doing so it also generated changes from previously pushed translations. It means that there have been many translation changes without generated files being generated.

So my question is: Should I or we (the community) make Issues/PR merely to push those generated files ? Because I am working on a PR but I feel like I shouldn't push those generated files in this PR because the changes are unrelated.

Another solutions might be to use GitHub Actions / Bots to automatically scan for edited translations ?

In any case thanks for reading.

**Comment by Tienisto** [comment]
There are some inconsistencies between the generated i18n files and their source files. I will fix this ASAP. I agree that the changes should not be included in the PR if they are unrelated. Thanks for noticing this issue!
